tests/integration/components/themed-syntax-test.js

```diff
@@ -49,3 +49,12 @@ test('it applies the "withBuffers" parameter', function(assert) {
   `);
   assert.ok(`${Ember.$(this.$().find('.line').first()).text()}${Ember.$(this.$().find('.line').last()).text()}` !== '', 'The buffers were left off');
 });
+
+test('it binds code', function(assert) {
+  this.set('code', 'first round');
+  this.render(hbs`{{themed-syntax code=code}}`);
```

Comment:
@pfmmfp If you have time, would you mind also adding an example block to the dummy site with the inline usage example?

Reply:
Added. Also updated README with the new parameter.